shared/actions/settings.js

```diff
@@ -66,6 +93,135 @@ function * saveNotificationsSaga (): SagaGenerator<any, any> {
   }
 }
 
+function * reclaimInviteSaga (invitesReclaimAction: InvitesReclaim): SagaGenerator<any, any> {
+  const {inviteId} = invitesReclaimAction.payload
+  try {
+    yield call(apiserverPostRpcPromise, {
+      param: {
+        endpoint: 'cancel_invitation',
+        args: [{key: 'invitation_id', value: inviteId}],
+      },
+    })
+    yield put(({
+      type: Constants.invitesReclaimed,
+      payload: undefined,
+      error: false,
```

Comment:
> I'm fine with making payload optional

Ah, looks like this isn't possible:

Marco: "Flow, I don’t think, has a notion of: If `P` is void, the key can be omitted."